webpack/webpack.common.js

```diff
@@ -35,7 +36,27 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        use: [MiniCssExtractPlugin.loader, "css-loader"],
+        use: [
+          MiniCssExtractPlugin.loader,
+          "css-loader",
+          {
+            loader: "postcss-loader",
+            options: {
+              ident: "postcss",
+              plugins: () => [
+                postcssPresetEnv({
+                  browsers: [
```

Comment:
We could probably revisit these versions... Maybe Firefox 52+, IE11, Safari 9, Chrome...75?80?. Not sure about Opera. Does that mean Chromium Opera or Presto opera?

Reply:
Good idea. I've looked at the documentation https://github.com/browserslist/browserslist#best-practices and there is "[defaults](https://browserl.ist)" list that we can use and possibly add things to it. 

So it can be `defaults, Firefox > 52, Safari >=9, Chrome >=75`
https://browserl.ist/?q=defaults%2C+Firefox+%3E+52%2C+Safari+%3E%3D9%2C+Chrome+%3E%3D75
And for Opera there is 64 and 63 already in the list, 63 has 0.01% usage.. so maybe that's enough?
And to your question on Chromium Opera or Presto, I think it depends on version number (I guess Presto has stopped at 12.18)

Reply:
LGTM!